imgui-examples/examples/support/mod.rs

```diff
@@ -105,6 +105,26 @@ pub fn run<F: FnMut(&Ui) -> bool>(title: String, clear_color: [f32; 4], mut run_
         update_mouse(&mut imgui, &mut mouse_state);
 
         let gl_window = display.gl_window();
+
+        let mouse_cursor = imgui.mouse_cursor();
+        if imgui.mouse_draw_cursor() || mouse_cursor == ImGuiMouseCursor::None {
+            // Hide OS cursor
+            gl_window.set_cursor_state(glutin::CursorState::Hide).unwrap();
+        } else {
+            // Set OS cursor
+            gl_window.set_cursor_state(glutin::CursorState::Normal).unwrap();
+            gl_window.set_cursor(match mouse_cursor {
+                ImGuiMouseCursor::None => unreachable!(),
```

Comment:
Could you add a message here as an argument to `unreachable!`? Something like "Mouse cursor was none"

Rust generates normal panic code for `unreachable!` and doesn't optimize the case away, so if we ever make a mistake here and end up in the unreachable case, it would be nice to have a good error message.